src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/router.ts

```diff
@@ -69,6 +69,11 @@ const router = createRouter({
       component: () => import("./components/pages/LogoutCallback.vue"),
       meta: { requiresAuth: false },
     },
+    {
+      path: "/application/summary",
```

Comment:
suggesting we go with `/manage-application`, the way it is now feels like a child route of application. Thoughts?